frontend/src/components/core/AdminButton.tsx

```diff
@@ -0,0 +1,22 @@
+import ArrowCircleRightIcon from '@mui/icons-material/ArrowCircleRight';
+import { useCachedMe } from '@hooks/useCachedMe';
+import * as React from 'react';
+
+
+interface AdminButtonProps {
+    adminUrl: string;
+    [key: string]: any;
+}
+
+export const AdminButton: React.FC<AdminButtonProps> = ({
+    adminUrl,
+    ...otherProps
+}) => {
+    const { data } = useCachedMe();
+    const isSuperUser = data.me.isSuperuser;
+
+    if (isSuperUser) {
```

Comment:
I think this logic should be on the backend, user should know about the admin panel only if is a super user